core-bundle/src/Controller/SitemapController.php

```diff
@@ -32,10 +32,12 @@
 class SitemapController extends AbstractController
 {
     private PageRegistry $pageRegistry;
+    private string $encoding;
 
-    public function __construct(PageRegistry $pageRegistry)
+    public function __construct(PageRegistry $pageRegistry, string $encoding = 'UTF-8')
```

Comment:
Why does this need to be configurable? We are not supporting other character sets than UTF-8, are we?

Reply:
This can be reverted if we implement my suggestion from https://github.com/contao/contao/pull/6148#discussion_r1233820757

Reply:
Then let‘s do this. 👍

Reply:
Should also be done in `SitemapEvent::addUrlToDefaultUrlSet()` then 😊

Reply:
that has only been added in Contao 5, see https://github.com/contao/contao/pull/6178